examples/PODT_phase_simulation/phase2D_3D.py

```diff
@@ -0,0 +1,57 @@
+# 3D partially coherent optical diffraction tomography (ODT) simulation
+# J. M. Soto, J. A. Rodrigo, and T. Alieva, "Label-free quantitative
+# 3D tomographic imaging for partially coherent light microscopy," Opt. Express
+# 25, 15699-15712 (2017)
```

Comment:
For comment like this you can append `#noqa` to tell flake8 that this should not be linted.

Reply:
Discussed in person, and there's no change needed here. Thanks.